tests/Handler/CreateOnlyTest.php

```diff
@@ -72,7 +72,7 @@ public function handleShouldExecuteTheCommandAndReturnAnEmptyResponse(): void
             StatusCodeInterface::STATUS_CREATED
         );
 
-        $request = new ServerRequest();
```

Comment:
Please add a new test scenario since we still have to cover the previous behaviour (where generated identifier is used).